torch/quantization/fx/_equalize.py

```diff
@@ -410,7 +496,12 @@ def scale_weight_node(
 
     # Multiply the weights row wise by the next equalization scale
     # Reshape the equalization scale so that we can multiply it to the weight along axis=0
-    next_equalization_scale_reshaped = reshape_scale(next_equalization_scale, 0, weight)
+    try:
+        next_equalization_scale_reshaped = reshape_scale(next_equalization_scale, 0, scaled_weight)
+    except RuntimeError:
+        raise ValueError(
+            "Input-weight equalization is unable to support general reshapes within models."
+        )
```

Comment:
Would be useful to add a more descriptive message here saying what the mismatch was